packages/db/src/schema.prisma

```diff
@@ -225,6 +229,8 @@ model Language {
   roles               LanguageMemberRole[]
   /// The pending import job if any.
   importJob           LanguageImportJob?
+  // The gloss history for this language.
+  glossHistory        GlossHistory[]
```

Comment:
I would move this up above `machineGlosses`

Reply:
Done.